src/main/java/app/controller/soundEngine/SoundBlockingBlock.java

```diff
@@ -0,0 +1,138 @@
+package app.controller.soundEngine;
+
+import app.controller.linAlg.Vector;
+import javafx.geometry.Rectangle2D;
+import javafx.scene.canvas.GraphicsContext;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+
+public class SoundBlockingBlock implements SoundFurniture{
```

Comment:
Potential refactor: the current Furniture should use an attribute that effects the sound, rather than there being another class that deals with sound.  That way we can add an attribute to the furniture and create seperate `SoundFurniture` `LightFurniture` `PhysicalFurniture`classes

See: https://refactoring.guru/design-patterns/decorator for more details